### DISCUSSION TITLE
@mantine/prism Rust Support

### DISCUSSION THREAD

**Discussion by JamalLyons** [mantinedev/mantine#4771] [Feature requests]
The `@mantine/prism` package does not support all the languages **prism** does by default. I'm not sure why, can this be fixed?

**Comment by rtivital** [comment]
You can find documentation on custom languages support here – https://github.com/FormidableLabs/prism-react-renderer#custom-language-support

**Comment by JamalLyons** [comment]
Hey i tried to add the custom language support for rust but it did not work. I tired the example from the repo:
```tsx
import { Prism as P } from 'prism-react-renderer';
//@ts-ignore
(typeof global !== 'undefined' ? global : window).Prism = P;
// eslint-disable-next-line @typescript-eslint/ban-ts-comment
// @ts-ignore
lazy(async () => await import('prismjs/components/prism-rust'));
```

*I'm rendering this in the same component I'm using the `@mantine/prism` package and `<Prism />` component*

and my packages are:

```json
"@mantine/prism": "^6.0.20",
"prism-react-renderer": "^2.0.6",
"prismjs": "^1.29.0"
```
